app/src/main/java/com/mapbox/vision/examples/CircleView.kt

```diff
@@ -0,0 +1,62 @@
+package com.mapbox.vision.examples
+
+import android.content.Context
+import android.graphics.*
+import android.graphics.Paint.ANTI_ALIAS_FLAG
+import android.util.AttributeSet
+import android.view.View
+import com.mapbox.vision.examples.utils.dpToPx
+
+class CircleView
+@JvmOverloads
+constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0
+) : View(context, attrs, defStyleAttr, defStyleRes) {
+
+    private val stokePx = context.dpToPx(2f)
```

Comment:
strokePx?

Reply:
yep

Reply:
I mean here is a typo, stroke and not stoke ^_^

Reply:
fix typo :)